.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -178,9 +178,9 @@ sqs_ListQueues:
         - sdk_version: 1
           github: rust_dev_preview/sqs
           excerpts:
-            - description:
+            - description: List & retrieve first &SQS; queue.

```

Comment:
List & retrieve -> List and retrieve
(These description are published into the documentation, so style is stricter.)